### ISSUE TITLE
Yet another Super Theater bug

### ISSUE THREAD

**Issue by BradyThe** [ParticleCore/Iridium#979]
- Iridium version: 2.1.0
- Browser: Firefox 125.0.1
- Troubleshoot Environment:
   - Default profile
   - Only Iridium add-on activated

This particular issue doesn't seem to be immediately related to #975 or #976, otherwise I probably wouldn't be here.

When activating the YouTube Theater Mode, while the Iridium Super Theater option is activated, the video player goes completely black, hiding the video and video controls.

Iridium Disabled/Super Theater Deactivated
![image](https://github.com/ParticleCore/Iridium/assets/8181555/8ab3b73a-3c7a-4438-a13d-1a4270f3fe86)

Super Theater Activated
![image](https://github.com/ParticleCore/Iridium/assets/8181555/5f656c67-b4d0-431d-b688-fcf87e9ebdc1)

Super Theater Activated, View scrolled
![image](https://github.com/ParticleCore/Iridium/assets/8181555/819ab98e-d37c-4a1e-a9e6-d9a92086ab53)

Out of curiosity, what is the difference between Super Theater and the native YouTube fullscreen scroll mechanic?

**Comment by ParticleCore** [comment]
Could you explain the exact steps that I need to follow in more detail? I am afraid that I cannot replicate this problem with the steps you've detailed:

![1](https://github.com/ParticleCore/Iridium/assets/9222661/c4b903aa-9334-446d-a1b5-968d7940a093)

---

> Out of curiosity, what is the difference between Super Theater and the native YouTube fullscreen scroll mechanic?

There shouldn't be any. Super Theater only allows the video to cover the full visible area of the browser (almost like a subset of fullscreen, but constrained within the browser), the rest stays the same. The difference is there are screen ratios (or simply just resizing the browser) that cause parts of the page to be visible, or just always visible by default (navigation bar, video details, suggested videos). This mode ensures the entire view is covered by the video when the page is at the top without having to go to fullscreen to get the same experience.

**Comment by ParticleCore** [comment]
Oh, I believe I know what is happening here.

Can you enable autoplay in your browser here and see if the issue still happens?
![image](https://github.com/ParticleCore/Iridium/assets/9222661/64270c53-39a2-4cf6-920c-c0380fa27ba0)

**Comment by BradyThe** [comment]
> Oh, I believe I know what is happening here.
> 
> Can you enable autoplay in your browser here and see if the issue still happens?

Well, this is embarrassing. That was indeed the problem (and now with the revival of Iridium a completely unnecessary setting).

**Comment by ParticleCore** [comment]
No, not at all, that is definitely a bug! I will address this one for the next version, it should not be black when the browser autoplay is disabled for sure, it is a scenario I did not account for. Thanks for bringing it to my attention!

**Comment by ParticleCore** [comment]
I know it's been a while, but I was trying to replicate this and it seems to no longer be an issue. At least I can't make it happen on my end with the following steps:

- Super Theater enabled in Iridium
- Autoplay enabled in Iridium
- Autoplay disabled in browser
- Toggle video theater mode


It is possible that the issue fixed itself after taking so long to address it, which would be very welcomed.

I will leave this open until release in case the issue is still existent, otherwise will be closed on release.
